universal-application-tool-0.0.1/app/modules/SecurityModule.java

```diff
@@ -101,175 +95,71 @@ protected void configure() {
     PlayCookieSessionStore sessionStore =
         new PlayCookieSessionStore(new ShiroAesDataEncrypter(aesKey));
     bind(SessionStore.class).toInstance(sessionStore);
-  }
 
-  @Provides
-  @Singleton
-  protected GuestClient guestClient(ProfileFactory profileFactory) {
-    return new GuestClient(profileFactory);
+    String applicantAuthClient = "idcs";
+
+    try {
+      applicantAuthClient = configuration.getString("auth.applicant_idp");
+    } catch (ConfigurationException ignore) {
+      // Default to IDCS.
```

Comment:
Just fyi, I had to bring this comment back because ErrorProne was complaining that "this exception should not be ignored" when I had a completely empty catch block

Reply:
Ah, looks like this is in internal/old exception that the public tool doesn't use.

https://github.com/google/error-prone/issues/1654